client/src/components/pages/LivePosts/index.js

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from 'react';
+import PostButton from 'components/utils/PostButton'
+import Post from 'components/utils/Post';
+import './style.css'
+
+class LivePosts extends Component {
+  state = {
+    livePosts: [],
+    error: '',
+  }
+
+  componentDidMount() {
+    //Mock Data for Testing the component//
+    const livePosts = [
+      { title: 'post 1', id: 1 },
+      { title: 'post 2', id: 2 },
+      { title: 'post 3', id: 3 },
+      { title: 'post 4', id: 4 },
+      { title: 'post 5', id: 5 },
+      { title: 'post 6', id: 6 },
+    ];
+    this.setState({ livePosts });
+  }
+
+  handleDelete = (id) => {
+    const { livePosts } = this.state;
+    //Testing Delete with Mock Data//
+    const deletedPost = livePosts.findIndex(post => post.id === Number(id));
+    this.setState((prevState) => {
+      prevState.livePosts.splice(deletedPost, 1);
+      return ({ livePosts: prevState.livePosts })
+    });
+  }
+
+  render() {
+    const { error, livePosts } = this.state;
+    return (
+      <section className='live-posts'>
+        {
+          error ?
+            <span className='live-posts-error'>{error}</span>
```

Comment:
@IsraaSulaiman There's a css trick you can do, if you want, to make this error jsx simpler:
First, you can always show the error in the jsx, instead of conditionally showing it.  So remove the ? :, and just always show it
`<span className='live-posts-error'>{error}</span>`
Next, you can use the `:empty` css pseudo-class to hide the error span when it's empty (in other words, when this.state.error is an empty string:
```
.live-posts-error:empty {
  display: none;
}
```
That makes the render function a little shorter and simpler (because it basically moves the show / hide logic into css), which is always a good thing because render functions always tend to get large and complicated so simplifying them always helps :)

Reply:
I like it 👍